### PR TITLE
fix(mgmt_api): return 503 when emqx is not running in `/status` (v4.3)

### PR DESCRIPTION
**Pull request by thalesmg**

### CODE REVIEW

File: apps/emqx_management/test/emqx_mgmt_api_SUITE.erl [hunk 1]
```diff
@@ -656,6 +680,51 @@ t_data_import_content(_) ->
     application:stop(emqx_rule_engine),
     application:stop(emqx_dashboard).
 
+t_status_ok(_Config) ->
+    {ok, #{ body := Resp
+          , status_code := StatusCode
+          }} = do_request(#{method => get, path => ["status"], headers => [],
+                                         body => no_body}),
```
Comment (zmstone) [inline apps/emqx_management/test/emqx_mgmt_api_SUITE.erl:687]:
```suggestion
    {ok, #{ body := Resp
          , status_code := StatusCode
          }} = do_request(#{method => get, path => ["status"], headers => [],
                            body => no_body}),
```

File: apps/emqx_management/test/emqx_mgmt_api_SUITE.erl [hunk 2]
```diff
@@ -656,6 +680,51 @@ t_data_import_content(_) ->
...
+      {match, _},
+      re:run(Resp, <<"emqx is running$">>)),
+    ?assertEqual(200, StatusCode),
+    ok.
+
+t_status_not_ok(_Config) ->
+    {ok, #{ body := Resp
+          , status_code := StatusCode
+          }} = do_request(#{method => get, path => ["status"], headers => [],
+                                         body => no_body}),
```
Comment (zmstone) [inline apps/emqx_management/test/emqx_mgmt_api_SUITE.erl:698]:
```suggestion
    {ok, #{ body := Resp
          , status_code := StatusCode
          }} = do_request(#{method => get, path => ["status"], headers => [],
                            body => no_body}),
```

File: apps/emqx_management/test/emqx_mgmt_api_SUITE.erl (hunk 1 above)
Comment (thalesmg) [reply apps/emqx_management/test/emqx_mgmt_api_SUITE.erl]: :heavy_check_mark:

File: apps/emqx_management/test/emqx_mgmt_api_SUITE.erl (hunk 2 above)
Comment (thalesmg) [reply apps/emqx_management/test/emqx_mgmt_api_SUITE.erl]: :heavy_check_mark:

File: changes/v4.3.22-en.md [hunk 3]
```diff
@@ -15,3 +15,5 @@
 - "Pause due to rate limit" log level demoted from warning to notice [#9134](https://github.com/emqx/emqx/pull/9134).
 
 - Restore old `emqx_auth_jwt` module API, so the hook callback functions registered in older version will not be invalidated after hot-upgrade [#9144](https://github.com/emqx/emqx/pull/9144).
+
+- Fixed the response status code for the `/status` endpoint.  Before the fix, it always returned 200 even if the EMQX application was not running.  Now it returns 503 in that case.
```
Comment (zmstone) [inline changes/v4.3.22-en.md:19]:
```suggestion
- Fixed the response status code for the `/status` endpoint [#9210](https://github.com/emqx/emqx/pull/9210). 
  Before the fix, it always returned `200` even if the EMQX application was not running.  Now it returns `503` in that case.
```

File: changes/v4.3.22-zh.md [hunk 4]
```diff
@@ -15,3 +15,5 @@
 - 限速 “Pause due to rate limit” 的日志级别从原先的 `warning` 降级到 `notice` [#9134](https://github.com/emqx/emqx/pull/9134)。
 
 - 保留老的 `emqx_auth_jwt` 模块的接口函数，保障热升级之前添加的回调函数在热升级之后也不会失效 [#9144](https://github.com/emqx/emqx/pull/9144)。
+
+- 修正了`/status`端点的响应状态代码。 在修复之前，它总是返回200，即使EMQX应用程序没有运行。 现在它在这种情况下返回503。
```
Comment (zmstone) [inline changes/v4.3.22-zh.md:19]:
```suggestion
- 修正了 `/status` API 的响应状态代码 [#9210](https://github.com/emqx/emqx/pull/9210)。
  在修复之前，它总是返回 `200`，即使 EMQX 应用程序没有运行。 现在它在这种情况下返回 `503`。
```

**Review by zmstone — APPROVED** [review]

**Pull request merged**